### PR TITLE
Skybox support

### PR DESCRIPTION
**Pull request by Shirkie01**
Added skybox support. Game will now choose random skybox from the Resources/Skyboxes folder on world creation and persist it to the world.xml file.

### CODE REVIEW

**Comment by svmnotn** [comment]
This has conflicts with master, please fix them

File: Assets/Scripts/Controllers/JobSpriteController.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ void OnJobCreated(Job job)
             //until the graphics system for tiles is fleshed out further
 
             job_go.transform.position = new Vector3(job.tile.X, job.tile.Y, 0);
-            sr.sprite = SpriteManager.current.GetSprite("Tile", "Empty");
+            sr.sprite = SpriteManager.current.GetSprite("Tile", "Solid");
```
Comment (svmnotn) [inline Assets/Scripts/Controllers/JobSpriteController.cs]: Is the `Solid` one 100% transparent? Also I don't see the need to change the name as technically that should still be empty space.

Comment (Shirkie01) [inline Assets/Scripts/Controllers/JobSpriteController.cs]: I modified the Empty tile to be transparent with outlines, but I needed another tile to be a solid color to retain the green job tile.

Comment (svmnotn) [inline Assets/Scripts/Controllers/JobSpriteController.cs]: Got it

**Comment by vogonistic** [comment]
Merged!

**Pull request merged**